NERODesign/content/MicrophoneComponent.qml

```diff
@@ -0,0 +1,112 @@
+
+
+/*
+This is a UI file (.ui.qml) that is intended to be edited in Qt Design Studio only.
+It is supposed to be strictly declarative and only uses a subset of QML. If you edit
+this file manually, you might introduce QML code that is not supported by Qt Design Studio.
+Check out https://doc.qt.io/qtcreator/creator-quick-ui-forms.html for details on .ui.qml files.
+*/
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Studio.Components
+
+Item {
+    id: item1
+    width: 1024
+    height: 768
+
+    Rectangle {
+        id: rectangle1
+        x: parent.width/2
+        y: parent.height/2
+        width: 20
```

Comment:
these should not be fixed. Everything should be in proportion to the parent height which is where this gets complicated and might be why an svg is better. You essentially have to create an aspect ratio for your component